docs/build/tools/other-paratimes/emerald/network.mdx

```diff
@@ -58,7 +58,7 @@ dedicated RPC endpoints, consider the following providers (in alphabetic order):
 
 | Name/Provider               | Mainnet URL                               | Testnet URL                               | EIP-3091 compatible |
 |-----------------------------|-------------------------------------------|-------------------------------------------|---------------------|
-| [Oasis Explorer][Oasis Protocol Foundation] | `https://explorer.oasis.io/mainnet/emerald` | `https://explorer.oasis.io/testnet/emerald` | Yes                 |
+| [Oasis Explorer][Oasis Protocol Foundation] | https://explorer.oasis.io/mainnet/emerald | https://explorer.oasis.io/testnet/emerald | Yes                 |
```

Comment:
These were deliberately made non-clickable so that people can copy&paste it to Metamask or other wallets when configuring a new network.